src/api/app/views/webui2/webui/package/_bottom_actions.html.haml

```diff
@@ -0,0 +1,56 @@
+%ul.list-inline.mb-0
+  - if @bugowners_mail.present? and @configuration['bugzilla_url']
+    %li.list-inline-item
+      = link_to(bugzilla_url(@bugowners_mail, "#{@project.name}/#{@package.name}: Bug"), class: 'nav-link') do
```

Comment:
Using Haml here instead of helpers makes the view more readable. [One example here](https://github.com/openSUSE/open-build-service/blob/a5265587132978ff1f2334f649b95fbfc789d189/src/api/app/views/webui2/webui/package/_tabs.html.haml#L4)